chalice/app.py

```diff
@@ -530,6 +530,13 @@ def on_sqs_message(self, queue, batch_size=1, name=None):
             registration_kwargs={'queue': queue, 'batch_size': batch_size}
         )
 
+    def on_cw_event(self, event_pattern, name=None):
```

Comment:
So I'm a little hesitant on using the name `on_cw_event`. Mainly because we recently rebranded the CloudWatch Events client in `botocore` to be EventBridge. And given EventBridge is a superset of the CloudWatch Event API, we may want to use event bridge functionality that gets launched in the future for this decorator. I'd prefer if we somehow generalize/abstract the name a bit more to remove the `cw` part of it. Maybe `event_pattern` or `patterned_event`? This will make it a little more consistent with the pre-existing CW event decorator`schedule()`.

@stealthycoin @jamesls Any thoughts about this?

Reply:
i'm open on any name suggestions here..   `on_bridge_event` ?

Reply:
I don't have a strong opinion on the name other than it's consistent with our naming convention of `on_<service>_<resource>`, so I guess that would make it `on_eb_event`, or `on_eventbridge_event`.

Reply:
Yeah consistency is the most important part for me. While it is a little redundant to have `event` twice in the name and a bit longer, I think I'd vote for `on_eventbridge_event` instead of `on_eb_event` because `eb` is more associated with `elasticbeanstalk` and I don't think `eb` is really an associated shortname to event bridge quite yet.

Reply:
i care much more about usability then consistency, ie. scheduled is already inconsistent by that nomenclature, and there's only like a handful of these methods. on_eventbridge_event feels overly verbose and repetitive (aka on event event ).. alternatively we could just keep on on_cw_event which would be less typing, or remove the duplication with on_bridge_event or on_eventbridge.. frankly the _resource suffix is redundant with the prefix `on_` since its always been `event`, which the prefix denotes.

Reply:
@jamesls what is your thoughts on `on_eventbridge`? I do agree with @kapilt that it is definitely redundant to have event twice in the name, especially given the service revolves around emitting events. Otherwise, I'm fine just making it `on_eb_event`. I really want to see us come up with a decision on this so we can proceed with the PR

Reply:
per discussion, going to continue with on_cw_event given its predominant extant usage. we may alias this in future to also support event bridge nomenclature.